core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ConfigResource.java

```diff
@@ -201,6 +201,39 @@ public Config getTopLevelConfig() {
     return config;
   }
 
+  @DELETE
+  @Operation(summary = "Deletes the specified Global-level compatibility level config and "
+      + "revert to the global default.", responses = {
+        @ApiResponse(content = @Content(
+            schema = @Schema(implementation = CompatibilityLevel.class))),
+        @ApiResponse(responseCode = "500", description = "Error code 50001 -- Error in the backend "
+          + "datastore")
+      })
+  public void deleteTopLevelConfig(
+      final @Suspended AsyncResponse asyncResponse,
+      @Context HttpHeaders headers) {
+    log.info("Deleting Global compatibility setting and reverting back to default");
+
+    Config deletedConfig;
+    try {
+      CompatibilityLevel currentCompatibility = schemaRegistry.getCompatibilityLevel(null);
+      Map<String, String> headerProperties = requestHeaderBuilder.buildRequestHeaders(
+          headers, schemaRegistry.config().whitelistHeaders());
+      schemaRegistry.deleteSubjectCompatibilityConfigOrForward(null, headerProperties);
```

Comment:
Lets change the following method names in KafkaSchemaRegistry (which are internal) since they now handle deletes for both global and subject level:

deleteSubjectCompatibilityConfigOrForward -> deleteCompatibilityConfigOrForward
deleteSubjectCompatibilityConfig -> deleteCompatibilityConfig
forwardDeleteSubjectCompatibilityConfigToLeader -> forwardDeleteCompatibilityConfigToLeader